### ISSUE TITLE
New option for optional bagspace

### ISSUE THREAD

**Issue by OpaReiter** [Jaliborc/Bagnon#1377]
**Is your feature request related to a problem? Please describe.**
The new 4 extra slots in the blizzard-bag don't have a function if you don't want the authenticator.
Also there is no hint what to do with these four bag-slots. They look like empty, but they can't be used

**Describe the solution you'd like**
An option where you could manage this: (x) Hide/show Blizzard-Bonus-Bags, 

**Additional context**
![grafik](https://user-images.githubusercontent.com/56296862/161960187-72249441-0b98-40e7-b6c1-bd03b4c82fc1.png)
![grafik](https://user-images.githubusercontent.com/56296862/161960492-279d59fc-31cf-4b57-96ac-6228d1b72d7a.png)

**Comment by Jaliborc** [comment]
They only should show up if you have an authenticator. They are showing up for you despite no authenticator?

**Comment by clancyby** [comment]
> They only should show up if you have an authenticator. Are they showing up for you despite no authenticator?

Yes, the authenticator is not connected, but the place is displayed and does not allow you to put anything there

**Comment by jamjamham** [comment]
I have the authenticator on, but no phonenumber, so the slot are still locked, never been able to see them while using bagnon, now suddenly there's 4 visible slots I can't use and no indicator that they're unusable. It's driving me crazy. An option to hide them would be appreciated.

**Comment by Elebato** [comment]
I never used the authenticator, but have exactly the same problem as described by @OpaReiter with Bagnon version 9.1.6.

**Comment by Dragonruler2020** [comment]
I don't have the authenticator, but I'm having the exact same problem @OpaReiter is having with Bagnon version 9.1.6.

**Comment by Jaliborc** [comment]
Guess I'll revert that change back. My bad, misunderstood what the authenticator API detection actually did.
